package.json

```diff
@@ -6,8 +6,10 @@
     "amp:validate": "wait-on http://localhost:7080/news/articles/c9rpqy7pmypo.amp && amphtml-validator http://localhost:7080/news/articles/c9rpqy7pmypo.amp",
     "audit:ci": "audit-ci -low",
     "bbcA11y": "bbc-a11y",
-    "build": "export NODE_ENV=production && rm -rf build && APP_ENV=test webpack && APP_ENV=live webpack",
-    "build:local": "rm -rf build && NODE_ENV=production webpack",
+    "build:ci": "export NODE_ENV=production && rm -rf build && npm run build:test && npm run build:live",
+    "build": "rm -rf build && cp env-config/local.env .env && NODE_ENV=production webpack",
```

Comment:
Cant comment on that line becuse github is dumb, but what about `npm run dev`, it doesnt appear to move env

Reply:
done